src/app/work-item/work-item-detail/work-item-detail.component.html

```diff
@@ -233,30 +233,37 @@ <h2 *ngIf="loggedIn && workItem.id" id="wi-detail-title-click" class="detail-tit
           <label class="col-sm-2">
             {{workItem.relationships?.baseType?.data?.attributes?.fields['system.iteration']?.label}}
           </label>
-          <div class="pull-left detail-iteration-name">
-            <span *ngIf="!searchIteration" (click)="lookupIterations()" class="pointer" id="iteration-assocaited">
+          <div class="combobox-container detail-iteration-name col-sm-6 padding-0">
+            <input type="hidden" name="" value="">
+            <span *ngIf="!searchIteration" (click)="lookupIterations()" class="pointer"
+            [class.none-assigned]="!workItem.relationships?.iteration?.data?.attributes?.name"
+             id="iteration-assocaited">
               {{ workItem.relationships?.iteration?.data?.attributes?.name || "None" }}
             </span>
-            <span *ngIf="searchIteration"
-              id="WI_details_iteration" dropdown>
-              <button class="btn btn-default" type="button" #dropdownButton
-                id="WI_details_iteration_dropdownbtn" data-toggle="dropdown" dropdownToggle>
```

Comment:
We need this ID to support our test cases, can you please add this ID in your new code!